src/graphql/queries/users.ts

```diff
@@ -1,17 +1,11 @@
 import gql from 'graphql-tag';
 
```

Comment:
Why can we not just keep the `AccessLevel` enum and define it in terms of numbers instead so that we can use it in conjunction with role ID?

```suggestion
export enum AccessLevel {
    COMMANDER = 1
    SUPERVISOR = 2
    DISPATCH = 3
}
```

Reply:
yeah this is actually a good suggestion, I'll try changing it to this. I changed them to be all numbers since I thought it would be more clear what the backend is accepting, but it's probably just more confusing